content/feature/Balance Sheet Implementation - Hellhound14.md

```diff
@@ -0,0 +1,36 @@
+|BALANCE SHEET IMPLEMENTATION|Date|
+|----------------------------|----|
+
+|Profit ($)| Loss ($)|Monthly Earnings ($)|Total ($)||
+|-------|---------|---------|--------|----|
+|576|543|33|33|2018-01-28|

```

Comment:
what is the date doing here?

Reply:
I don't know what to do here....
The example.md file doesn't help much